content/articles/adding-custom-methods-for-linq-queries/index.md

```diff
@@ -0,0 +1,161 @@
+### Adding custom methods for LINQ queries (C#) 
+### Introduction
+By introducing extension methods to the IEnumerableT> interface, you may expand the set of methods available for LINQ queries. Any data sequence can be converted into one single value by creating your custom aggregate method in SQL Server. 

```

Comment:
change "Any data sequence can be converted into one single value by creating your custom aggregate method in SQL Server. " to "You can convert any data sequence into one value by creating your custom aggregate method in SQL Server. "